src/MediaCardGroup.js

```diff
@@ -42,18 +43,32 @@ const MediaCardGroup = React.createClass({
     },
 
     render() {
-        let children = React.Children.map(this.props.children,
+        const { stagger } = this.props;
+
+        const children = React.Children.map(
+            this.props.children,
             (child) => React.cloneElement(child, {
                     onExpand: this.onExpand.bind(this, child),
                     isExpanded: this.state.expanded === child,
-                })
-            );
+                }
+            )
+        );
+
+        const renderList = stagger ? (
+                <Stagger
+                    transition="MediaCard__animation"
+                    delay={70}
+                >
+                    { children }
+                </Stagger>
+        ) : children;
```

Comment:
Yeah, that is more readable. Thanks.

There are a bunch of refactors that I'd like to do like this in another PR soon. And will certainly keep in mind writing ternaries with JSX in the future.